compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -2638,11 +2634,13 @@ private ParameterizedTypeInfo(BType paramValueType, int index) {
 
     private BSymbol resolveClientDeclPrefix(BSymbol symbol) {
         Location location = symbol.pos;
-        if (!this.clientDeclarations.containsKey(location)) {
+        Map<Location, PackageID> clientDeclarations = symTable.clientDeclarations;
```

Comment:
Codecov warnings can be ignored for now. Tests can be added only after merging the project API changes.